### PR TITLE
audit entry stub

### PR DESCRIPTION
**Pull request by demoth**
add some basic stubs to create audit coding

### CODE REVIEW

**Comment by Adonai** [comment]
I'm simple man, I see PR, I hit "Approve"

File: dto/src/main/java/com/dao/mydebts/dto/AuditLogResponse.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import com.dao.mydebts.entities.Actor;
+import com.dao.mydebts.entities.Debt;
+
+import java.util.List;
+
+public class AuditLogResponse {
+    private Actor me;
+
+    private List<Debt> debts;
```
Comment (Adonai) [inline dto/src/main/java/com/dao/mydebts/dto/AuditLogResponse.java]: Why debts in response, I thought it should be AuditEntry?

Comment (demoth) [inline dto/src/main/java/com/dao/mydebts/dto/AuditLogResponse.java]: yes

Comment (demoth) [inline dto/src/main/java/com/dao/mydebts/dto/AuditLogResponse.java]: fixed

**Pull request merged**